docs/Tutorial.md

```diff
@@ -16,6 +16,9 @@ npm start
 
 You should be up and running with an empty React application on port 3000.
 
+The API is not stable yet, so make sure you read the correct version of this tutorial,
+which you should be able to find locally as `node_modules/admin-on-rest/docs/Tutorial.md`.
+
```

Comment:
Moved it to the end of the §, it should be more natural to take a break at this point.